pkg/build/admission/admission.go

```diff
@@ -35,7 +35,7 @@ type buildByStrategy struct {
 // on policy based on the build strategy type
 func NewBuildByStrategy(client client.Interface) admission.Interface {
 	return &buildByStrategy{
-		Handler: admission.NewHandler(admission.Create),
+		Handler: admission.NewHandler(admission.Create, admission.Update),
```

Comment:
is it odd that we're gating create and update on create permission on the subresource? I'd sort of expect create to check create, and update to check update